deployments/overlays/nodeport/kustomization.yaml

```diff
@@ -1,7 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
 resources:
 - ../../base
 # Uncomment to add custom css as a configmap
 # - ../../css 
 
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+images:
+- name: ghcr.io/openmicrotools/jwt-inspect
+  newTag: latest
+
+metadata:
+  annotations:
+    name: jwt-inspect
+  labels:
+    app: jwt-inspect
+namespace: open-microtools
```

Comment:
I feel like we don't need from line #12 to #17. If we want to have commonLabels and commonAnnotations we can have it defined in deployment/base folder.

Reply:
Having at least namespace here allows users to make a single change to change all the namespaces, same for the annotations/labels. We can do it either way, but I thought having it here made it easier for the user.

Reply:
similar as the other thread, I don't know if user will ever want to apply Ingress in a different namespace than base defined where service is. and I think ingress should have the same common labels and annotations as base folder.